README.md

```diff
@@ -43,19 +33,14 @@ To run the test suite:
 bundle exec rake
 ```
 
-## Populating data
-
-If you are a GOV.UK developer using the development VM, you can [run the replication script to populate the database](docs/import_production_data.md).
-
-### Run ETL processes locally
+## Further documentation
 
-- To run the ETL process locally, you need to  [set up Google Analytics credentials in development](docs/google_analytics_setup.md).
+- [Internal API documentation](./docs/api)
+- [Importing production data](docs/import_production_data.md)
```

Comment:
I think we can just remove this one, as it's part of the central GOV.UK Docker docs. I don't think anyone maintains the `govuk data` tool, and it should be redundant since we already have one way of replicating stuff.